### PR TITLE
Related Posts: add inline docs for all actions and filters.

### PR DESCRIPTION
**Pull request by jeherve**

### CODE REVIEW

**Comment by retrofox** [comment]
Nice PR. Just like a side comment you can add hyphen before the description. But it's just FYI.

File: modules/related-posts/jetpack-related-posts.php [hunk 1]
```diff
@@ -495,12 +509,36 @@ public function get_for_post_id( $post_id, array $args ) {
 			'exclude_post_ids' => array(),
 		);
 		$args = wp_parse_args( $args, $defaults );
+		/**
+		 * FIlter the arguments used to retrieve a list of Related Posts.
```
Comment (retrofox) [inline modules/related-posts/jetpack-related-posts.php]: F`I`lter in upper case

**Comment by jeherve** [comment]
> you can add hyphen before the description.

Good feedback, it is indeed more readable.

However, we chose to follow [Core's standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters) for all inline docs. I think it's best if we stick to that, until Core changes their standards.

File: modules/related-posts/jetpack-related-posts.php (hunk 1 above)
Comment (jeherve) [inline modules/related-posts/jetpack-related-posts.php]: Thanks, fixed in 44169f216c220aa227f5e310c8a8e10b8d03ad6b

**Comment by dereksmart** [comment]
Nice one.  Didn't realize RP has so many filters :)

**Pull request merged**